src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -30,17 +30,18 @@ export default (
       return { ...state, languages: action.params.languages }
     case 'PUSH_CATEGORY':
       if (state === null) {
-        throw Error('Cannot push category on not initialized cityContent')
+        return null /* If you are on the landing page, then the `state` is null. If you quickly navigate
+                       to Landing screen while data is being fetched, then this line is reached. */
       }
       return pushCategory(state, action)
     case 'PUSH_EVENT':
       if (state === null) {
-        throw Error('Cannot push event on not initialized cityContent')
+        return null
       }
       return pushEvent(state, action)
     case 'MORPH_CONTENT_LANGUAGE':
       if (state === null) {
-        throw Error('Cannot morph content language on not initialized cityContent')
+        return null
       }
```

Comment:
The same should apply to `FETCH_CATEGORY_FAILED`, `FETCH_RESOURCES_FAILED` and probably also to `SWITCH_CONTENT_LANGUAGE_FAILED`.